frontend/model/contracts/chatroom.js

```diff
@@ -163,9 +163,7 @@ sbp('chelonia/defineContract', {
       }
     },
     'gi.contracts/chatroom/join': {
-      validate: actionRequireInnerSignature(objectOf({
-        memberID: optional(string) // user id of joining memberID
-      })),
```

Comment:
Why was `actionRequireInnerSignature` removed from here?

Reply:
Oh, nice catch. I should revert it.